OWML.ModHelper.Menus/LayoutManager.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.UI;
+using OWML.Common.Menus;
+using System.Collections.Generic;
+using OWML.ModHelper.Events;
+
+namespace OWML.ModHelper.Menus
+{
+    public class LayoutManager : ILayoutManager
+    {
+        private const int FontSize = 36;
+        private static readonly Vector2 NormalPivot = new Vector2(0.5f, 0.5f);
+
+        public LayoutGroup LayoutGroup { get; private set; }
+        public int ChildCount => LayoutGroup.transform.childCount;
+
+        private readonly UIStyleManager _styleManager;
+        private readonly ModUIStyleApplier _styleApplier;
+        private readonly Vector3 _scale;
+        private readonly HashSet<Graphic> _constantGraphics = new HashSet<Graphic>();
+        private readonly HashSet<Graphic> _backingGraphics = new HashSet<Graphic>();
```

Comment:
This hashset doesn't get any values added to it. Can it be removed?

Reply:
now it gets values added. Not sure why I missed that